chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertifiedProductController.java

```diff
@@ -861,6 +871,32 @@ public void streamSEDDetailsDocumentContents(final HttpServletResponse response)
                 throw new ValidationException(pcpDto.getErrorMessages(), pcpDto.getWarningMessages());
             }
 
+            if (pendingCp.getDeveloper() != null && pendingCp.getCertifyingBody() != null
```

Comment:
I'm pretty sure the CertifyingBody of a pendingCp can never be null, and if it was I don't think we'd even get this far. That said, I'm not sure it's actually wrong to perform the check. Up for discussion

The second part here (and related to the next line) is related to what happens if a Listing is being uploaded for a new Developer. In that case, there won't be a Developer in the system to check against, and can't be until the Listing is confirmed. I think this part might warrant a group discussion just so we're all on the same page as to what that means

Reply:
On the first part, I'd say unless we are 100% sure then the check should be made. But, I'm not dead set against removing the checks at all. You know the software better than me. My perspective is simply that I didn't know for sure so I checked.

As for what happens if we create a new developer - I would like to discuss that as well.